setup.py

```diff
@@ -113,6 +113,7 @@
     "SQLAlchemy==1.4.49 ",
     # Pin Flask-SQLAlchemy version due to apply_driver_hacks
     "Flask-SQLAlchemy==3.0.0",
+    "Flask-Alembic==2.0.1",
```

Comment:
is there an issue with this? If so, maybe add a comment to the relevant issue so we can unpin it when it is solved?